test/Microsoft.AspNet.Mvc.Core.Test/ApplicationModel/DefaultActionModelBuilderTest.cs

```diff
@@ -285,6 +286,48 @@ public void IsAction_StaticMethods(string methodName)
         }
 
         [Fact]
+        public void BuildActionModel_EnableCorsAttributeAddsCorsAuthorizationFilter()
+        {
+            // Arrange
+            var corsOptions = new CorsOptions();
+            corsOptions.AddPolicy("policy", new CorsPolicy());
+            var mockOptions = new Mock<IOptions<CorsOptions>>();
+            mockOptions.SetupGet(o => o.Options)
+                       .Returns(corsOptions);
+            var builder = new DefaultActionModelBuilder(null, corsOptions: mockOptions.Object);
+            var typeInfo = typeof(EnableCorsController).GetTypeInfo();
+            var method = typeInfo.GetMethod("Action");
+
+            // Act
+            var actions = builder.BuildActionModels(typeInfo, method);
+
+            // Assert
+            var action = Assert.Single(actions);
+            Assert.True(action.Filters.Any(f => f is CorsAuthorizationFilter));
```

Comment:
Is this the most correct assertion? Generally `Assert.True` is frowned upon because the failure messages are very poor - there's probably a more specific assertion that could be done. But either way, should this be checking that there's exactly one filter of the type we want as opposed to see if there are 1 or more?